test/unit/handlers/authenticate-handler_test.js

```diff
@@ -93,6 +94,43 @@ describe('AuthenticateHandler', function() {
     });
   });
 
+  describe('validateAccessToken()', function() {
+    it('should fail if token has no valid `accessTokenExpiresAt` date', function() {
+      var model = {
+        getAccessToken: function() {}
+      };
+      var handler = new AuthenticateHandler({ model: model });
+
+      var failed = false;
+      try {
+        handler.validateAccessToken({
+          user: {}
+        });
```

Comment:
You can get rid of the `failed` variable and just put `should.fail()` inside the try block.

```
try {
  handler.validateAccessToken({
    user: {}
  });
  should.failed();
} catch (err) {
// ...
```

Makes the test case a bit easier to read.